cmake/Modules/AdocMan.cmake

```diff
@@ -44,7 +44,13 @@ find_program(ADOCCOMMAND_PATH
 )
 
 if(NOT EXISTS ${ADOCCOMMAND_PATH})
-  message(WARNING "AsciiDoc processor not found, man pages will not be generated. Install asciidoctor or use the CMAKE_PROGRAM_PATH variable.")
+  set(ADOC_MISSING_MSG "AsciiDoc processor not found, man pages will not be generated. Install asciidoctor or use the CMAKE_PROGRAM_PATH variable.")
+
+  if (ENABLE_DOC STREQUAL "Auto")
```

Comment:
Will this work correctly if ENABLE_DOC was set in another case, like -DENABLE_DOC="auto" ?

Reply:
It appears that no, STREQUAL is case-sensitive.
https://cmake.org/pipermail/cmake/2012-June/050651.html

Meh, things were easier in autotools, why did rnp switch to cmake :D